doc/rst/technotes/remote.rst

```diff
@@ -111,10 +104,10 @@ of a host array into it:
 
    on here.gpus[0] var GpuA = A[GpuPortion];
 
-If the target locale is a GPU locale, and the initialization expression
-is either a promoted expression or a loop expression, it is transformed into
-a GPU kernel as it would be in an ``on`` block. Thus, the following line of
-code generates a kernel launch:
+Since the initialization expression is executed on the target locale,
+if the array is initialized with a GPU-eligible expression, this expression
+is transformed into a GPU kernel as it would be in an ``on`` block.
+Thus, the following line of code generates a kernel launch:
```

Comment:
I understand that this is not part of this PR, but would it make sense to add link(s) to the GPU technote and/or the blog series right before "Implementation Details" below?

Reply:
The GPU tech note is linked just above, so I'm inclined to leave this as is.